Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -190,7 +193,14 @@ private List<(EntityUid Entity, CargoPalletComponent Component, TransformCompone
                 continue;
             }
 
+            if (((requestType & BuySellType.Sell) != 0 && (comp.PalletType & BuySellType.Sell) == 0) ||
+                ((requestType & BuySellType.Buy) != 0 && (comp.PalletType & BuySellType.Buy) == 0))
+            {
```

Comment:
```suggestion
            if ((requestType & comp.PalletType) == 0)
            {
```

Reply:
That... makes a lot more sense, yeah. Ha.